build/templates/errors.rst.mako

```diff
@@ -43,6 +43,22 @@ ${helper.get_rst_header_snippet('DriverNotInstalledError', '-')}
 
         An error due to using this module without the driver runtime installed.
 
+${helper.get_rst_header_snippet('DriverTooOldError', '-')}
+
+    .. py:currentmodule:: ${module_name}.errors
+
+    .. exception:: DriverTooOldError
+
+        An error due to using this module with an older version of the driver runtime.
```

Comment:
You can use `${driver_name}` to be explicit about the driver runtime.

Reply:
This has been addressed by #1764.